project-management/FujitsuMeetingDates

```diff
@@ -0,0 +1,14 @@
+## Meetings Descriptions
```

Comment:
Can we add a more general heading to the top of the file?

# Fujitsu/Turing Sprint Meetings

## Meeting descriptions